### ISSUE TITLE
"Not here" error message when opening report fields

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#38252]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.51-3
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @Ollyws 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1710320724430419

## Action Performed:
Preconditon : Have some report fields for the collect workspace in OldDot
1. Open app
2. Sign in as the employee
3. Create an expense report
4. Opent the expense report
5. Click on the report fields

## Expected Result:
Should not lead to not here page

## Actual Result:
getting a not here page when opening report fields

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/38435837/225e9ba9-7b9d-427c-b279-550776fac5b0

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by hungvu193** [comment]
Our BE has updated and this one will be solved when this PR is merge
https://github.com/Expensify/App/pull/36553
So let hold this on https://github.com/Expensify/App/pull/36553

**Comment by bernhardoj** [comment]
Oh, there is an issue created for this. How about we fix this first separately to unblock https://github.com/Expensify/App/pull/37583?

@hungvu193 @thienlnam

**Comment by hungvu193** [comment]
Base on this [comment](https://github.com/Expensify/App/pull/37583#issuecomment-1995205765) I think we still need to hold this one. But let see what's his thought 😄

**Comment by adelekennedy** [comment]
@JmillsExpensify I think this falls under wave control release one

**Comment by thienlnam** [comment]
As mentioned, this App PR should fix things https://github.com/Expensify/App/pull/36553 along with the one back-end PR that should be merged soon

**Comment by adelekennedy** [comment]
Should this just be put on hold then until then? Or are we confident enough to close it?

**Comment by thienlnam** [comment]
Either works, I would just link to that issue so we don't re-create this

**Comment by adelekennedy** [comment]
then I'm closing it! We'll reopen if it's an issue
